sensio/sphinx/codeblock.py

```diff
@@ -0,0 +1,49 @@
+from docutils.parsers.rst import Directive, directives
+from docutils import nodes
+from sphinx.util.nodes import set_source_info
+
+"""
+The only change with the built-in CodeBlock class that comes with Sphinx
+is that linenos is always on.
```

Comment:
that's not true. this is the only change compared to Sphinx 1.2. But compared to Sphinx 1.3, it also removes all new features